sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -275,6 +275,7 @@ private[sql] class JDBCRDD(
    */
   private def compileFilter(f: Filter): String = f match {
     case EqualTo(attr, value) => s"$attr = ${compileValue(value)}"
+    case EqualNullSafe(attr, value) => s"$attr <=> ${compileValue(value)}"
```

Comment:
Do you think this should be treated differently for each different dialect ( maybe only for MySQL ) or should this be closed?